utils/results/store_bids_tree.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+
+log = logging.getLogger(__name__)
+
+def store_bids_tree(context, run_label, destination_id):
```

Comment:
Not used?  This was stuff for testing what .metadata.json things actually worked.  Did you want to do something with it here?

Reply:
I moved the method into "fly>dev_helpers.py", in case we need it again.